python/tests/physical.py

```diff
@@ -34,7 +34,7 @@ def radiating_base(self, sq_ratio, solve_cw=True):
 
         w = 0.30
 
-        s = mp.structure(self.gv, one, mp._pml(self.ymax / 3.0))
+        s = mp.structure(self.gv, one, mp.pml(self.ymax / 3.0))
```

Comment:
@ChristopherHogan, should this test be using `mp.PML` (i.e. the high-level object) nowadays?

Reply:
This is a low level test, so rewriting the whole thing in the high level API is on my todo list. You can leave it as is for now.